CONTRIBUTING.md

```diff
@@ -51,6 +51,14 @@ Below is a short excerpt of some points in there:
 
 See the nixpkgs manual for more details on [standard meta-attributes](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes).
 
+## First Time contributors
+
+You will need to add yourself to [maintainers](https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers)
```

Comment:
You do **not** need to for pass by contributions. Adding your self as a maintainer means that you want to take over long term responsibility for a "package" or "module".

Thing is, that you need to add yourself to the maintainers file in a separate commit from the changes that will reference yourself the first time.

This way, the changes to the package or module can be reverted without removing the contributor from the maintainers file, which in worst case might break other packages where they are maintainers in.

Reply:
## Adding self as Maintainer

You only need to add yourself as a [maintainer](https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers) if you want to take long term responsibility for a package or module. Otherwise, you can make pass by contributions without being a maintainer.

To add yourself as a maintainer, you need to make a separate commit in the maintainers file before you reference yourself in the changes to the package or module.

This way, if the changes to the package or module are reverted, you will still remain in the maintainers file and avoid breaking other packages or modules where you are a maintainer.

Reply:
I rephrased what you said into command, if that looks good.